plugins/services/src/js/constants/OperatorTypes.js

```diff
@@ -1,38 +1,55 @@
+const DEFAULT_HELP = "A string, integer or regex value.";
+const OPTIONAL_HELP = `${DEFAULT_HELP} This field is optional.`;
```

Comment:
I'm not sure if we should keep these two texts here, you also have it as default below and we can calculate if a field ist optional or not 🤔 

I will add texts here as well in my PR and I already felt bad adding these, IMHO this file should be as clean/small as possible.